src/WebJobs.Script/Description/Node/Script/functionTemplate.js

```diff
@@ -5,6 +5,9 @@ var f = require('{0}'),
     util = require('util');
 
 return function (context, callback) {{
+    // TEMP HACK: workaround for https://github.com/tjanczuk/edge/issues/325
+    setImmediate(() => {{}});
```

Comment:
Yep, it's probably safer to leave in the old hack as well until the edge library is fixed.  We could try to remove it for a major release, but there's a chance it could break uncommon customer scenarios.